src/index.js

```diff
@@ -0,0 +1,14 @@
+import _ from 'lodash';
+import './style.css';
+import AddScores from './views/AddScore.js';
+import Footer from './views/Footer.js';
+import Modal from './views/Modal.js';
+import RecentScores from './views/RecentScores.js';
+
+const leaderContent = document.querySelector('.leaderboard-content');
+
+// update leaderboard content
+leaderContent.innerHTML = _ + RecentScores() + AddScores();
```

Comment:
- [ ] See the error I get when I run your app.

  ![image](https://user-images.githubusercontent.com/36542986/209448319-13e7377e-9343-421e-b24d-fa54eb4f9440.png)

  You can fix this by importing `recentScores` correctly and using it as intended.

  ```
  import { recentScores } from './views/RecentScores.js';
  
  ... 
  
  // update leaderboard content
  leaderContent.innerHTML = _ + recentScores() + AddScores();
  ```

Reply:
## Thanks @TedLivist for your comment, now it's fixed based on the recommendation. 😃

Reply:
- Hello @habasm, I am still experiencing the same issues experienced by the previous reviewer, please follow the advice of the previous reviewer 🙏 
<img width="960" alt="ythrth" src="https://user-images.githubusercontent.com/73607512/209450026-8a039234-a9ff-4f93-b422-22ed09af0aca.PNG">